java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2800,4 +2803,90 @@ public void objectRefcount(BaseClient client) {
         assertEquals(OK, client.set(key, "").get());
         assertTrue(client.objectRefcount(key).get() >= 0L);
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void geoadd(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, GeospatialData> membersToCoordinates =
+                new java.util.HashMap<>(
```

Comment:
```suggestion
```

Reply:
`Immutable object is modified` you get this warning when applying a put operation on it. I just created a hashmap instead of this wrapping.